### PR TITLE
Updating the testrun status via MQTT.

### PR DESCRIPTION
**Pull request by hitnik**

### CODE REVIEW

File: framework/python/src/common/session.py
Comment (sofyakurilova) [inline framework/python/src/common/session.py:41]: Should it be /events/status?

Comment (hitnik) [reply framework/python/src/common/session.py:41]: According to notes from Bi-weekly sync, it was proposed to call this topic "status"

Comment (jboddey) [reply framework/python/src/common/session.py:41]: Correct yes, as this is not necessarily an event, it should just be could /status.

File: framework/python/src/common/mqtt.py
Comment (jboddey) [inline framework/python/src/common/mqtt.py:46]: Should this line be duplicated?

File: framework/python/src/common/session.py
Comment (jboddey) [inline framework/python/src/common/session.py:383]: Is there a reason why it would not be updated in the check above? Why not just use this part standalone?

**Comment by jboddey** [comment]
@hitnik could you provide an example output of the JSON that is sent with this?

File: framework/python/src/common/mqtt.py
Comment (hitnik) [reply framework/python/src/common/mqtt.py]: removed

File: framework/python/src/common/session.py
Comment (hitnik) [reply framework/python/src/common/session.py]: fixed

**Comment by hitnik** [comment]
> @hitnik could you provide an example output of the JSON that is sent with this?

Same as the /system/status response.

**Comment by jboddey** [comment]
@hitnik Just one pylint issue to resolve and I will try this out

File: framework/python/src/common/session.py
Comment (jboddey) [inline framework/python/src/common/session.py:55]: The JSON format is not correct. fastapi automatically converts some Python objects into JSON for us. Now that it is not used, raw Python objects are being included in the response. See here: https://paste.googleplex.com/6069255923826688 Before this can be merged, more work is required to verify the JSON being received by the UI.

**Review by jboddey — CHANGES_REQUESTED** [review]

File: framework/python/src/common/session.py
Comment (hitnik) [reply framework/python/src/common/session.py:55]: I fixed it in the ui branch

**Comment by jboddey** [comment]
Looks good now. However, we will merge this is once v1.4 is released

**Pull request merged**
